### PR TITLE
BDI-39 Login Form

### PR DESCRIPTION
**Pull request by timrebergen**
Working changes to the token storage

### CODE REVIEW

File: bdi-frontend/src/views/RegisterComponent.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template id="app-template">
   <v-container width="700">
     <v-card class="mx-auto px-6 py-8" width="600">
-      <v-card-title class="text-primary" align="center">
+      <v-card-title class="" style="color: #4FC6EC; font-weight: 700; font-size: 36px" align="center">
```
Comment (jafetsinke) [inline bdi-frontend/src/views/RegisterComponent.vue:4]: Shouldn't use inline styles in favour of vuetify classes

File: bdi-frontend/src/views/RegisterComponent.vue [hunk 2]
```diff
@@ -90,8 +90,8 @@ export default {
         password: ''
       },
       template: '#app-template',
-      show1: false,
-      show2: false,
+      show1: true,
```
Comment (jafetsinke) [inline bdi-frontend/src/views/RegisterComponent.vue:93]: Show/hide button does not work, also getting `[Vue warn]: Property "show1" was accessed during render but is not defined on instance. `

File: .idea/WeatherProject.iml
Comment (jafetsinke) [inline .idea/WeatherProject.iml:1]: Probably want to add .idea to gitignore

File: bdi-frontend/src/App.vue [hunk 3]
```diff
@@ -2,6 +2,8 @@
   <router-view />
 </template>
 
-<script lang="ts" setup>
-  //
+<script>
```
Comment (jafetsinke) [inline bdi-frontend/src/App.vue:5]: not sure why this change was made? belongs to commit "almost_functional_login"

File: bdi-frontend/src/appRouter.js
Comment (jafetsinke) [inline bdi-frontend/src/appRouter.js:1]: Aren't we using 2 routers at the same time this way? router/index.ts and appRouter.js

File: bdi-frontend/src/store/auth.js
Comment (jafetsinke) [inline bdi-frontend/src/store/auth.js:1]: Request is working!

File: bdi-frontend/src/views/RegisterComponent.vue [hunk 4]
```diff
@@ -90,8 +90,8 @@ export default {
         password: ''
       },
       template: '#app-template',
-      show1: false,
-      show2: false,
```
Comment (jafetsinke) [inline bdi-frontend/src/views/RegisterComponent.vue:94]: password is shown by default

File: bdi-frontend/src/views/LoginComponent.vue [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        ></v-text-field>
+
+        <v-text-field
+          v-model="userCredentials.password"
+          prepend-icon="mdi-lock"
+          :append-icon="show1 ? 'mdi-eye' : 'mdi-eye-off'"
+          :type="show1 ? 'text' : 'password'"
+          hint="Minimaal 8 tekens"
+          counter
+          @click:append="show1 = !show1"
```
Comment (jafetsinke) [inline bdi-frontend/src/views/LoginComponent.vue:28]: Show/hide is not working. suggest rename variable name aswell

File: bdi-frontend/src/views/LoginComponent.vue [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+          color="primary"
+          variant="underlined"
+          label="Wachtwoord"
+          placeholder="Wachtwoord invullen"
+          required
+        ></v-text-field>
+        <v-btn
+          type="submit"
+          class="btn btn-primary"
+          color="#4FC6EC"
```
Comment (jafetsinke) [inline bdi-frontend/src/views/LoginComponent.vue:38]: color can take "primary" instead of HEX

File: bdi-frontend/src/views/LoginComponent.vue [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+          size="large"
+          variant="elevated"
+          style="color: white; font-weight: 700"
```
Comment (jafetsinke) [inline bdi-frontend/src/views/LoginComponent.vue:41]: Prevent using inline styles

**Review by jafetsinke — CHANGES_REQUESTED** [review]

**Comment by timrebergen** [comment]
Fixed the stuffies

File: bdi-frontend/.gitignore
Comment (jafetsinke) [inline bdi-frontend/.gitignore:1]: nice

File: bdi-frontend/src/views/RegisterComponent.vue (hunk 2 above)
Comment (jafetsinke) [reply bdi-frontend/src/views/RegisterComponent.vue:93]: Vue warn is fixed! show/hide button not working

**Review by jafetsinke — CHANGES_REQUESTED** [review]

**Review by jafetsinke — APPROVED** [review]

**Pull request merged**